packages/app/src/cli/models/extensions/schemas.ts

```diff
@@ -39,6 +39,8 @@ export const BaseExtensionSchema = schema.define.object({
     .optional(),
   metafields: schema.define.array(MetafieldSchema).optional().default([]),
   categories: schema.define.array(schema.define.string()).optional(),
+  authenticatedRedirectStartUrl: schema.define.string().optional(),
+  authenticatedRedirectRedirectUrls: schema.define.array(schema.define.string()).optional(),
```

Comment:
@Safi1012 Hey,

I was just rebasing with main and saw this change.  This change seems to make `authenticatedRedirectStartUrl` and `authenticatedRedirectRedirectUrls` part of the schema for every extension.  I assume it's only needed for the customer_accounts extension?  

2 questions: 

1. Could these be defined only on the `customer_accounts` schema instead?
2. Do these properties need to make it as far Dev Console? I ask because they were added to the JSON data that the extensions dev console recieves.

Reply:
Hey @byrichardpowell,

I'm super sorry! You are absolutely right, they just need to be part of the `customer_accounts` schema. Thank you for the comment!

1. yes
2. Good point, it's not needed inside of the Dev Console 👍
3. I'm not 100% sure what do you mean by settings here in this context? What I want is that every `customer_accounts` extension is able to be deployed with those two new optional properties. Is that possible with settings?
 
I'll create a PR tomorrow morning and tag you there, since it's already after hours for me here 🙈 I hope that's okay 🙏

Reply:
Cool cool, no need to be sorry.  We should setup the CLI so that the extensions team are auto included on changes like this.  I'll add that to my todo list.

Yep, tomorrow is totally fine. If you want to pair lemme know